src/locales/ja-JP/stream.ftl

```diff
@@ -0,0 +1,895 @@
+  .title = 取り消し戦
+
+comments-rte-spoiler = ネタバレ
+
+comments-rte-sarcasm = 辛辣
+
+comments-rte-externalImage =
+  .title = 外部画像
+
+comments-remainingCharacters = あと { $remaining }  文字
+
+comments-postCommentFormFake-signInAndJoin = サインインして、会話に参加する
+
+comments-postCommentForm-rteLabel = コメントを投稿する
+
+comments-postCommentForm-rte =
+  .placeholder = { comments-postCommentForm-rteLabel }
+
+comments-postCommentFormFake-rte =
+  .placeholder = { comments-postCommentForm-rteLabel }
+
+comments-replyButton-reply = 返信
+comments-replyButton =
+  .aria-label = {$username} でコメントに返信する
+
+comments-permalinkViewQuery-storyNotFound = { comments-streamQuery-storyNotFound }
+
+comments-replyCommentForm-submit = 送信
+comments-replyCommentForm-cancel = キャンセル
+comments-replyCommentForm-rteLabel = 返信を書く
+comments-replyCommentForm-rte =
+  .placeholder = { comments-replyCommentForm-rteLabel }
+
+comments-commentContainer-threadLevelLabel = スレッドレベル { $level }：
+comments-commentContainer-highlightedLabel = ハイライト：
+comments-commentContainer-ancestorLabel = 上位：
+comments-commentContainer-replyLabel =
+   { $username } から返信 <RelativeTime></RelativeTime>
+comments-commentContainer-questionLabel =
+   { $username }  から質問 <RelativeTime></RelativeTime>
+comments-commentContainer-commentLabel =
+   { $username } からコメント <RelativeTime></RelativeTime>
+comments-commentContainer-editButton = 編集
+
+comments-commentContainer-avatar =
+  .alt = { $username } のアバター
+
+comments-editCommentForm-saveChanges = 変更を保存する
+comments-editCommentForm-cancel = キャンセル
+comments-editCommentForm-close = 閉じる
+comments-editCommentForm-rteLabel = コメントを編集
+comments-editCommentForm-rte =
+  .placeholder = { comments-editCommentForm-rteLabel }
+comments-editCommentForm-editRemainingTime = 編集可能時間： <time></time> まで
+comments-editCommentForm-editTimeExpired = 編集可能時間が終了しました。 このコメントは編集できなくなります。 別のコメントを投稿してみませんか？
+comments-editedMarker-edited = 編集済み
+comments-showConversationLink-readMore = この会話をもっと見る>
+comments-conversationThread-showMoreOfThisConversation =
+  このコメントをもっと見る
+
+comments-permalinkView-currentViewing =
+comments-permalinkView-singleConversation =
+comments-permalinkView-youAreCurrentlyViewing =
+  現在ひとつの会話を表示しています。
+comments-inReplyTo = <Username></Username> に返信する
+comments-replyingTo = <Username></Username> に返信する
+
+comments-reportButton-report = 報告
+comments-reportButton-reported = 報告済み
+comments-reportButton-aria-report =
+  .aria-label = {$username} のコメントを報告する
+comments-reportButton-aria-reported =
+  .aria-label = 報告済み
+
+comments-sortMenu-sortBy = 並び替え
+comments-sortMenu-newest = 最新
+comments-sortMenu-oldest = 最も古い
+comments-sortMenu-mostReplies = もっと返信を見る
+
+comments-userPopover =
+  .description = より多くのユーザー情報を含むポップオーバー
+comments-userPopover-memberSince = { DATETIME($timestamp, year: "numeric", month: "long", day: "numeric") } 以降のメンバー
+comments-userPopover-ignore = 無視する
+
+comments-userIgnorePopover-ignoreUser = {$username} を無視しますか？
+comments-userIgnorePopover-description =

```

Comment:
It looks like this translation is English. Did you mean to include a Japanese translation here?